README.md

```diff
@@ -14,15 +14,15 @@ Pull requests are welcome. Please, create an issue first explaining what you wan
 
 ### Development
 
-You'll need `git`, `yarn`, `docker` and `docker-compose`.
+You'll need `git`, `yarn`, `docker` and the Docker Compose plugin.
```

Comment:
I would actually drop the name `docker-compose-plugin` here. That's what it's called for fedora anyway. Had to find it in the docs, as I could never make `docker compose` without a dash work.

https://docs.docker.com/engine/install/fedora/#set-up-the-repository

Reply:
Hmm, I'm a bit confused tbh. :thinking: 

> That's what it's called for fedora anyway.

In most Linux distros, the package is called `docker-compose-plugin` I believe.

<details>
<summary><code>$ apt list docker-compose-plugin</code></summary>

```
Listing... Done
docker-compose-plugin/bookworm,now 2.21.0-1~debian.12~bookworm amd64 [installed]
N: There are 7 additional versions. Please use the '-a' switch to see them.
```

</details>

> as I could never make `docker compose` without a dash work.

Assuming you followed the docs you shared in the past few months, I believe that would install Docker Compose v2. So, a plugin for Docker rather than a separate utility.

If you wouldn't mind, would you be willing to share the output of:

```sh
docker --version
docker-compose --version
docker compose version
```

Reply:
Sorry, I think I wasn't very clear. It works fine now.

I did have to dig through the docs to get it installed earlier. Before that I wasn't aware, that `docker-compose` is v1 and `docker compose` is v2 and you need to install something separate. So every time anyone came up with commands without a dash, it was very confusing why it was working for them but not for me.

After running ` sudo dnf install docker-compose-plugin` earlier today it just started working. So I think we should name that package in the readme, to give a hint. As just saying `Docker Compose plugin` might send most people to a search engine.  Saying `docker-compose-plugin` might tip of some of the more experienced users.

Reply:
Ahh, this all makes sense, thanks for clarifying!
I can relate actually, I had the same experience for a few months too. ^-^'

Making the change now.